ballerina-shell/README.md

```diff
@@ -0,0 +1,130 @@
+# Ballerina Shell
+
+A REPL program for the [ballerina language](https://github.com/ballerina-platform/ballerina-lang). Ballerina is an open source programming language and platform for cloud-era application programmers to easily write software that just works.
+
+The Ballerina-shell tool is an interactive tool for learning the Ballerina programming language and prototyping Ballerina code. Ballerina-shell is a Read-Evaluate-Print Loop (REPL), which evaluates declarations, statements, and expressions as they are entered and immediately shows the results. Currently, the tool is run via the command line. Using Ballerina-shell, you can enter program statements one at a time and immediately see the result.
+
+## Demo
+
+![Recording](./docs/demo.gif)
+
+## Modules
+
+The project is implemented in two base modules.
```

Comment:
```suggestion
The project is implemented in three base modules.
```
?

Reply:
Typo. Previously it was only 2 modules and I think I forgot to update it. I'll fix this.